### PR TITLE
Feature/chatroom

### LINKED ISSUE #17: In addition to an asynchronous messaging thread I want the ability to communicate with others through a chat room/hangout room similar to that of Discord or Google Hangouts
-13pt

Josh Bender

### PR DESCRIPTION
**Pull request by jb1685**
This feature works similarly to iMessage as a user to user chatroom

Test files are located in /chatroom/test.py
Test instructions are located in the root directory Readme.md

Closes: #17

### CODE REVIEW

File: chatroom/models.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class Chat(models.Model):
+    to_id = models.CharField(max_length=50)
+    from_id = models.CharField(max_length=50) 
+
+class Message(models.Model):
+    chat = models.ForeignKey(Chat, on_delete=models.CASCADE)
+    message = models.CharField(max_length=5000)
```
Comment (Squirrelofudun) [inline chatroom/models.py:10]: this might work better as a TextField()

**Review by Squirrelofudun — APPROVED** [review]
left a comment about TextField vs CharField, otherwise looks fine

**Review by ryanhummel — APPROVED** [review]
Everything looks good

**Comment by jb1685** [comment]
fudged up the thingy

**Comment by jb1685** [comment]
deleted sqlite3 db

**Pull request closed without merging**